internal/controller/util/json_util.go

```diff
@@ -108,17 +108,25 @@ func getResourcesList(k8sClient client.Reader, hook *kubeobjects.HookSpec) ([]cl
 	}
 
 	if hook.NameSelector != "" {
+		log.Info("getting resources using nameSelector", "nameSelector", hook.NameSelector)
+
 		objsUsingNameSelector, err := getResourcesUsingNameSelector(k8sClient, hook, objList)
 		if err != nil {
+			log.Error(err, "error getting resources using nameSelector", "error", err)
```

Comment:
Instead of logging the error here, I would prefer to just prefix the error message with a string to say it failed for the name selector.